Language/Variables/Variable Scope & Qualifiers/volatile.adoc

```diff
@@ -77,16 +78,19 @@ void blink() {
 }
 ----
 
+Use an atomic block for atomic access to a 16-bit `int`:
```

Comment:
A couple ideas for improving this description:
- Instead of "Use an atomic block for", say "Use `ATOMIC_BLOCK to ensure". The reason being that I've never seen the term "atomic block" used before and we don't want to train people to use non-standard vocabulary.
- Instead of "a 16-bit `int`", say "a variable with size greater than the microcontroller's 8-bit data bus".